vnext/Microsoft.ReactNative.Cxx.UnitTests/Microsoft.ReactNative.Cxx.UnitTests.vcxproj

```diff
@@ -8,11 +8,13 @@
     <VCProjectVersion>15.0</VCProjectVersion>
     <ProjectGuid>{6c60e295-c8ca-4dc5-b8be-09888f58b249}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
-    <RootNamespace>Microsoft_ReactNative_Cxx_UnitTests</RootNamespace>
+    <RootNamespace>Microsoft.ReactNative</RootNamespace>
     <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.18362.0</WindowsTargetPlatformVersion>
-    <WindowsTargetPlatformMinVersion>10.0.17134.0</WindowsTargetPlatformMinVersion>
+    <WindowsTargetPlatformMinVersion>10.0.15063.0</WindowsTargetPlatformMinVersion>
```

Comment:
>15063 [](start = 42, length = 5)

why is there a decrease in versions?

Reply:
This is a minimal supported version of WIndows. We must support 15063.0 in order to have C++/WinRT projects load .Net assemblies. In this test project it was not needed, but I want to make sure that tests are compiled with the same settings as other projects.